### PR TITLE
try adding learnchef to the matrix

### PR DESCRIPTION
**Pull request by lamont-granquist**
Signed-off-by: Lamont Granquist <lamont@scriptkiddie.org>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -252,6 +252,25 @@ matrix:
...
+    before_install:
+      - gem update --system $(grep rubygems omnibus_overrides.rb | cut -d'"' -f2)
+      - gem install bundler -v $(grep bundler omnibus_overrides.rb | cut -d'"' -f2)
+    bundler_args: --without changelog development docgen guard integration maintenance omnibus_package tools aix bsd mac_os_x solaris windows --frozen
+    before_script:
+      - sudo iptables -L DOCKER || ( echo "DOCKER iptables chain missing" ; sudo iptables -N DOCKER )
+      - cd kitchen-tests
+      - git clone https://github.com/learn-chef/awesome_customers_ubuntu cookbooks/awesome_customers_ubuntu
+    script:
+      - bundle exec kitchen test fedora-23
```
Comment (lamont-granquist) [inline .travis.yml]: yeah i'm not even doing anything with fedora this PR that was copypasta that has been modified and is gone now

**Comment by lamont-granquist** [comment]
@tas50 is there a centos-6 issue in the chef-client cookbook in the 7.0.0 release?  picked up the new version when i bumped berks in this PR, and had to back down to make centos-6 green again.

**Comment by tas50** [comment]
Ohai isn't detecting init_package in docker. It's on my to-do for tomorrow.

**Comment by lamont-granquist** [comment]
closing this PR to leave the windows work, got stuck on kitchen-appbundle-updater not working on windows i think...

**Pull request closed without merging**